javascript-modal/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript-modal</title>
+  <link href="styles.css" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <div>
+      <button class="modal-button" type="button">Open Modal</button>
+    </div>
+  </div>
+  <div>
+    <dialog class="modal-box">
```

Comment:
While this works, try completing the functionality without the `dialog` element using a standard div